### DISCUSSION TITLE
More than one year - fans are still waiting!

### DISCUSSION THREAD

**Discussion by Kepalu** [emoon/HippoPlayer#209] [General]
Hey! It's been more than one year since the last update and fans are still waiting!

I still use HippoPlayer but one thing that is needed soooo desperately is the time bar! So you can see the time stamp of each track you play and you can rewind it to any time you want.

I really hope this project isn't abandoned? It was one of the most promising things on Earth!

My hopes are high!

**Comment by emoon** [comment]
Hey,

The project isn't abandoned and a new version is being worked on. That said it's a spare time project that has to fit in with other projects as well. So while progress is being slow right now, rest assured a new release will come at some point.

**Reply by illettante** [reply]
I can't code but if you have an example of look and feel for the gui and need some assets generated i could maybe contribute pixels or vectors?

**Reply by emoon** [reply]
Thanks! That sounds great, I hope to get back to the project within a few months and I will update this thread then.

**Reply by joonas-foo** [reply]
Could you  make binary packages of the player, .deb or flatpak?-) Building it myself seems a bit adventurous with my skills. This project seems really interesting, I use Strawberry at this moment and also find it excellent.

**Reply by emoon** [reply]
Hey, Yes. The plan is to make an image for Linux once I get back to it. I'm not sure exactly what to use just yet, but it will be something like Flatpak or perhaps AppImage

**Reply by joonas-foo** [reply]
I wrote a few feature suggestions to Strawberry's page, would you be interested of them? Will you have a plugin interface for Hippoplayer? What about an option to choose the oldskool Amiga view of the player? :-) Anyway, here's the Strawberry suggestions thread: https://forum.strawberrymusicplayer.org/topic/389/a-few-features-that-i-would-be-happy-of-deadhead-plugin-support-for-discogs-support/2 .

**Reply by emoon** [reply]
The player is fully built around plugins. Pretty much everything is a plugin that can be swapped out except some specific core components, but UI, players, etc are all plugins. While there are some interesting suggestions in that thread, they would likely be a bit lower priority in my case. The player I'm building is mostly focusing on oldskool formats such as mods,xm,sid:s etc and while flac,mp3 is supported it's not a "focus" for it. A oldskool Amiga view is something I have considered and will likely be implemented at some point.

**Reply by joonas-foo** [reply]
Ok! If you need a beta tester, I'm available for that. Used to betatest Amiga's Filemaster back in the nineties and was quite good in finding bugs. Still find bugs in many programs I use :-).

**Reply by emoon** [reply]
Thanks! I'm still in the process of moving over to a new (custom written) UI system and it takes time, but in the end it will be worth it. I will update once it's ready for testing, but it isn't there yet.
